ssn/chapters/Actuation.html

```diff
@@ -177,27 +186,31 @@ <h5><a href="#SOSAActuationCollection">sosa:ActuationCollection</a></h5>
             The following consistency rules with respect to the Actuation properties listed above:
             <br><br>
```

Comment:
It is *far* better to use `<p>...</p>` than `<br><br>...<br><br>`. LIkewise, lists should use `<ol>` or `<ul>` with `<li>`. Such markup items should be used when available, rather than using poor-man's HTML and faking them.

Reply:
Unrelated to this PR. 
Please create a separate issue.